tests/smoke-gradle-version-catalog.yaml

```diff
@@ -0,0 +1,1172 @@
+                      requirement: 1.2.0
+                      source: null
+                  version: 1.2.0
+                - name: junit:junit
+                  requirements:
+                    - file: gradle/libs.versions.toml
+                      groups: []
+                      metadata:
+                        property_name: jUnitVersion
+                      requirement: 4.13.2
+                      source: null
+                  version: 4.13.2
+                - name: androidx.test.ext:junit
+                  requirements:
+                    - file: gradle/libs.versions.toml
+                      groups: []
+                      metadata:
+                        property_name: androidxJunit
+                      requirement: 1.1.3
+                      source: null
+                  version: 1.1.3
+                - name: androidx.compose.ui:ui-test-junit4
+                  requirements:
+                    - file: gradle/libs.versions.toml
+                      groups: []
+                      metadata:
+                        property_name: composeUi
+                      requirement: 1.2.0
+                      source: null
+                  version: 1.2.0
+                - name: androidx.compose.ui:ui-tooling
+                  requirements:
+                    - file: gradle/libs.versions.toml
+                      groups: []
+                      metadata:
+                        property_name: composeUi
+                      requirement: 1.2.0
+                      source: null
+                  version: 1.2.0
+                - name: androidx.compose.ui:ui-test-manifest
+                  requirements:
+                    - file: gradle/libs.versions.toml
+                      groups: []
+                      metadata:
+                        property_name: composeUi
+                      requirement: 1.2.0
+                      source: null
+                  version: 1.2.0
+                - name: androidx.test.espresso:espresso-core
+                  requirements:
+                    - file: gradle/libs.versions.toml
+                      groups: []
+                      metadata:
+                        property_name: espresso
+                      requirement: 3.5.0
+                      source: null
+                  version: 3.5.0
+                - name: org.jmailen.kotlinter
+                  requirements:
+                    - file: gradle/libs.versions.toml
+                      groups:
+                        - plugins
+                      requirement: 3.11.0
+                      source: null
+                  version: 3.11.0
+            dependency_files:
+                - /build.gradle
+                - /settings.gradle
+                - /gradle/libs.versions.toml
+                - /app/build.gradle
+    - type: create_pull_request
+      expect:
+        data:
+            base-commit-sha: 99ad8dcb667e058158bbd8448e0cccd1453b1f59
```

Comment:
These `base-commit-sha`s should match the one in the input: `943c1eaeb4ed9383ec77c0ffe95ec727c1f7c5e6`

Reply:
Yup, I'm working on fixing this and a bunch of other issues now :)